src/Models/SiteTreeLink.php

```diff
@@ -134,4 +134,19 @@ public function getTitle(): ?string
         // Use page title as a default value in case CMS user didn't provide the title
         return $page->Title;
     }
+
+    public function getLinkTypeDefaultTitle(): string
+    {
+        $pageExist = $this->Page()->exists() && $this->getURL();
+
+        // If the page doesn't exist, we can't get the title
+        if (!$pageExist) {
+            return _t(
+                'LinkField.DEFAULT_LINK_TITLE_WRONG_SITE_TREE_LINK',
+                'Page missing',
```

Comment:
```suggestion
                static::class . '.MISSING_DEFAULT_TITLE',
                'Page missing'
```
Use `static::class` instead of link field - it's standard practice for our localisation strings to be based on a class.
And use `MISSING_DEFAULT_TITLE` for the localisation string since that's nice and generic and can be shared across all link implementations.

Reply:
Updated